### ISSUE TITLE
Transfer maintainership to oauthlib organization?

### ISSUE THREAD

**Issue by pope1ni** [lepture/flask-oauthlib#380]
Hi @lepture,

It looks as though your focus is now on [authlib](https://github.com/lepture/authlib/) and that you are no longer maintaining flask-oauthlib. Would you consider handing over maintainership to the recently established [oauthlib](https://github.com/oauthlib/) organization on github - if they'd be happy to have it?

This would be helpful to those stuck using flask-oauthlib for now and it seems like a sensible new home...

**Comment by lepture** [comment]
Yeah. That would be possible. But there seems no suitable person for this project yet from what I know in https://github.com/lepture/flask-oauthlib/graphs/contributors 

If there are volunteers to take this project, they can make an announcement in the PR, and I will review the PR to know that if he/she can take the job. I will spare some time at the end of this year to process this issue.

**Comment by aparamon** [comment]
@lepture Currently this project has a number of appealing PRs, e.g. https://github.com/lepture/flask-oauthlib/pull/325, https://github.com/lepture/flask-oauthlib/pull/384.
Do you welcome rebasing them against [authlib](https://github.com/lepture/authlib) to be merged there sooner?
